src/nummeth.cpp

```diff
@@ -235,7 +235,89 @@ int secant(							// find x given f(x) (secant method)
     }
 	return i;
 }			// ::secant
+
 //-----------------------------------------------------------------------------
+int secant( // screen secant success; report calcuation if failure
```

Comment:
I don't understand the strategy here.  There should be high-level explanatory comments.  "Try classic secant method.  If it fails converge, do x, y, z in an effort to allow execution to continue."

Reply:
I was told to add code to repeat `secant` upon failure with text output. I have expanded some comments to clarify.